### ISSUE TITLE
Consider dropping '.js' prefix in Node.js require statements

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/chipper#535]
See https://github.com/phetsims/yotta/issues/37#issuecomment-270572333.

For instance, instead of
```js
var googleAnalytics = require( './googleAnalytics.js' );
```

we should ideally have 
```js
var googleAnalytics = require( './googleAnalytics' );
```

which should work due to Node.js saying:

> If the exact filename is not found, then Node.js will attempt to load the required filename with the added extensions: .js, .json, and finally .node.

**Comment by jonathanolson** [comment]
This is beneficial because then we don't need to disable eslint for the line.

Tagging for developer meeting for awareness and yay/nay for me (or someone) to fix these instances across the codebase.

**Comment by samreid** [comment]
Yes, we should drop the ".js" and corresponding `//eslint-disable` workarounds.

**Comment by jbphet** [comment]
Yes, go for it!
